test/check-exports/test.mjs

```diff
@@ -0,0 +1,109 @@
+import { spawnSync } from 'child_process'
+import { deepStrictEqual } from 'assert'
+import { dirname, resolve } from 'path'
+import { fileURLToPath } from 'url'
+
+const cwd = dirname(fileURLToPath(import.meta.url))
+const hook = resolve(cwd, '..', '..', 'hook.mjs')
+
+let errored = false
+
+function getExports (name, loader) {
+  const args = ['--input-type=module', '--no-warnings', '-e', `import * as lib from '${name}'; console.log(JSON.stringify(Object.keys(lib)))`]
+  if (loader) args.push(loader)
+  const out = spawnSync(process.execPath, args, { cwd })
+  if (out.status !== 0) {
+    console.error(out.stderr.toString())
+    throw new Error()
+  }
+  const stdout = out.stdout.toString()
+  return JSON.parse(stdout).sort()
+}
+
+function testLib (name) {
+  try {
+    const expected = getExports(name)
+    const actual = getExports(name, `--experimental-loader=${hook}`)
+    deepStrictEqual(actual, expected, `Exports for ${name} are different`)
+    console.log(`✅  Exports for '${name}' match`)
+  } catch (err) {
+    console.error(`❌  Error getting exports for '${name}':`, err)
```

Comment:
For debugging it would be nice to get the version of the package installed. What do you think about logging out the version via `npm list`?

```
spawnSync('npm', ['list', name], { cwd })
```